en/docs/get-started/about-this-release.md

```diff
@@ -56,6 +56,16 @@ It is now available to download from [here](https://github.com/wso2/product-apim
 - Defaulting code and configs to do with event based
```

Comment:
We need to add the Vhost feature together with details under the new feature section as well.

Reply:
updated